l3.c

```diff
@@ -20,6 +34,9 @@ struct l3_entry
     uint64_t    arg2;
 };
 
+/**
+ * L3 Log Structure definitions:
+ */
```

Comment:
Add small prologs to each struct describing what it stands for.

Please correct or add more descriptive text if you wish to see some.